cmd/xcodeUITests.go

```diff
@@ -21,8 +21,7 @@ import (
 
 var xcodeUITestsCmd = &cobra.Command{
 	Use:   "xcodeuitests",
-	Short: "Xcode project scanner for UI tests",
-	Long:  `Scan an Xcode project for UI test targets`,
```

Comment:
The `Long` value has some meaningful information of what this command does. I like the `Short` value update but do not we want to keep the `Long` one somehow?

Reply:
Short is the command description when the command is listed as a subcommand. Long is the main description (Short gets used as the main description if Long is not provided).

Updated the Short and Long descriptions, now it looks like:

```
$ codesigndoc scan -h

Scans the project's code signing settings,
and exports the required code signing files.

Usage:
  codesigndoc scan [command]

Available Commands:
  xcode        Scans the project's code signing settings for Xcode Archive and IPA export actions
  xcodeuitests Scans the project's code signing settings for Xcode Build For Testing action

Flags:
...
```

and 

```
$ codesigndoc scan xcode -h

Scans the project's code signing settings for Xcode Archive and IPA export actions
and exports the required code signing files.

Usage:
  codesigndoc scan xcode [flags]

Flags:
...
```

and 

```
$ codesigndoc scan xcodeuitests -h

Scans the project's code signing settings for Xcode Build For Testing action
and exports the required code signing files.

Usage:
  codesigndoc scan xcodeuitests [flags]

Flags:
...
```